MelonJs.JavaScript/Bindings/Tools/load.js

```diff
@@ -1,4 +1,85 @@
-﻿const load = (path) => {
-    const content = fs.read(path);
-    melon_internal_script_injector(content);
+﻿const load = (path, options = { useHttpRequest: true, useUnsafeInjectorLoader: false }) => {
```

Comment:
why `useHttpRequest ` is set as true by default?

Reply:
It depends when it is being used, but fs would be cool too